libs/ng-lib/tsconfig.lib.prod.json

```diff
@@ -4,6 +4,6 @@
     "declarationMap": false
   },
   "angularCompilerOptions": {
-    "enableIvy": true
```

Comment:
just checking, this is removed as from v13 and up this is by default true?

Reply:
Yes, done by the nx schematics. not a manual thing